source/client/stream_decoder.cc

```diff
@@ -156,7 +156,7 @@ void StreamDecoder::setupForTracing() {
   // We pass in a fake remote address; recently trace finalization mandates setting this, and will
   // segfault without it.
   const auto remote_address = Envoy::Network::Address::InstanceConstSharedPtr{
-      new Envoy::Network::Address::Ipv4Instance("127.0.0.1", 0)};
+      new Envoy::Network::Address::Ipv4Instance("127.0.0.1")};
```

Comment:
Note to self/reviewers: this is a no-op, but will avoid breaking on the next Envoy
update.